### PR TITLE
Swift 3.0 support

### PR DESCRIPTION
**Pull request by gkaimakas**

### CODE REVIEW

File: GaugeKit/CAShapeLayer+oval.swift [hunk 1]
```diff
@@ -11,109 +11,112 @@ import Foundation
...
+	                   anchorPoint: CGPoint? = nil
+		) -> CAShapeLayer {
+
+		var arc = CAShapeLayer()
+		let path = CGMutablePath()
+		let Y = bounds.midY
+
+		path.move(to: CGPoint(x: lineWidth, y: Y))
+		path.addLine(to: CGPoint(x: bounds.width-lineWidth, y: Y))
+//		CGPathMoveToPoint(path, nil, lineWidth, Y)
```
Comment (skywinder) [inline GaugeKit/CAShapeLayer+oval.swift:33]: Why you comment it out?

File: GaugeKit/Gauge+Animations.swift [hunk 2]
```diff
@@ -9,15 +9,15 @@
 import UIKit
 
 extension Gauge {
-    public func animateRate(duration: NSTimeInterval, newValue: CGFloat, completion: (Bool) -> ()) -> Void {
+    public func animateRate(_ duration: TimeInterval, newValue: CGFloat, completion: @escaping (Bool) -> ()) -> Void {
```
Comment (skywinder) [inline GaugeKit/Gauge+Animations.swift:12]: Are you sure, that escaping is needed there? What for?

**Review by skywinder — COMMENTED** [review]
Waiting for your reply

File: GaugeKit/Gauge+Animations.swift (hunk 2 above)
Comment (gkaimakas) [reply GaugeKit/Gauge+Animations.swift:12]: You are keeping a reference to `completion` on `animationCompletionBlock` later on.

When keeping references of closures the @escaping is required

File: GaugeKit/CAShapeLayer+oval.swift (hunk 1 above)
Comment (gkaimakas) [reply GaugeKit/CAShapeLayer+oval.swift:33]: `CGPathMoveToPoint` and `CGPathAddLineToPoint`were changed in Swift 3.0
They are now member functions of `CGPath`, one called `move(to:)` and the other `addLine(to:)`

I should have just deleted the commented code though

File: GaugeKit.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -186,9 +186,13 @@
 				TargetAttributes = {
 					1C9404CF1B31CF85001BF122 = {
 						CreatedOnToolsVersion = 6.3.1;
+						DevelopmentTeam = 96LJ93WQBU;
```
Comment (skywinder) [inline GaugeKit.xcodeproj/project.pbxproj:189]: You don't have to put your dev team id in open source project

File: GaugeKit/Gauge+Animations.swift (hunk 2 above)
Comment (skywinder) [reply GaugeKit/Gauge+Animations.swift:12]: agreed.

**Comment by skywinder** [comment]
Thanks for the fixes. But I prefer to update by Xcode project by myself. All changes in `master` branch already. Feel free to merge it and add suggestions how to improve the code. 👍

**Pull request closed without merging**